api/src/pcapi/routes/backoffice/offer_edition_rules/forms.py

```diff
@@ -0,0 +1,57 @@
+from flask_wtf import FlaskForm
+import wtforms
+
+from pcapi.core.categories import categories
+from pcapi.core.categories import subcategories_v2
+
+from ..forms import fields
+from ..forms import utils
+
+
+class SearchRuleForm(FlaskForm):
+    class Meta:
+        csrf = False
+
+    category = fields.PCSelectMultipleField(
+        "Catégories", choices=utils.choices_from_enum(categories.CategoryIdLabelEnum)
+    )
+    subcategory = fields.PCSelectMultipleField(
+        "Sous-catégories", choices=utils.choices_from_enum(subcategories_v2.SubcategoryIdEnumv2)
```

Comment:
Pourquoi permettre toutes les sous-catégories dans le filtre, si on ne peut avoir que les catégories `not is_event` à la création, et donc dans la liste ?

Surtout qu'Elisa avait écrit « on oublie les filtres parce que ça simplifie » :slightly_smiling_face:

Reply:
C'est aussi simple de ne pas trier ici 😅 D'autant qu'un jour, on aura peut-être aussi les autres sous-cat
Mais les filtres sont déjà fait, donc on peut les garder